### DISCUSSION TITLE
messages & gallery & sharedr

### DISCUSSION THREAD

**Discussion by jekv2** [FossifyOrg/General-Discussion#255] [Q&A]
I'm on androud 11, sharedr works on my android 11 but fails to operate by selecting a photo sent through messages and opened through gallery.

https://sharedr.rejh.nl/

one screenshot sharedr fails through messages then gallery.

one screenshot showing an image opened through just gallery and sharedr works.
![Screenshot_20241130-085500](https://github.com/user-attachments/assets/c07c5cbf-2820-4ab5-acbc-51a32d99427e)
![Screenshot_20241130-090200](https://github.com/user-attachments/assets/c129656e-a6a0-4ee2-a4fc-a96b35985dd8)

**Comment by Aga-C** [comment]
We aren't going to add support for the deprecated app.
